src/plugins/explore/public/components/query_panel/components/editor_stack/editor_stack.tsx

```diff
@@ -46,11 +51,12 @@ const EditorStack: React.FC<EditorStackProps> = ({
   return (
     <div className="queryPanel__editorStack" data-test-subj="queryPanelEditorStack">
       <PromptEditor
-        onChange={onPromptChange}
+        isPromptLoading={isPromptLoading}
```

Comment:
not sure if we are prop drilling, but prop drilling is technically an anti pattern https://www.geeksforgeeks.org/reactjs/what-is-prop-drilling-and-how-to-avoid-it/ and could lead to performance issues due to unneccessary renders